typedlistadapter/src/test/java/pl/allegro/typedlistadapter/ExampleUnitTest.kt

```diff
@@ -0,0 +1,18 @@
+package pl.allegro.typedlistadapter
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
this test is unnecessary

Reply:
I know, but this is a good reminder that this library needs some tests :) And CI needs to run something in test stage, so I prefer to leave it for now